src/components/composer/composer/composer.ts

```diff
@@ -338,7 +346,13 @@ export class Composer extends Component<ComposerProps, SpreadsheetChildEnv> {
   }
 
   onKeydown(ev: KeyboardEvent) {
-    let handler = this.keyMapping[ev.key] || this.keyCodeMapping[ev.code];
+    if (ev.key.startsWith("Arrow")) {
```

Comment:
It may be a better way, but I personnaly think that it was more readable to have all the handler together instead of some processors defined in the handlers variables above and the arrow-key processor directly called here :/

Reply:
I personally prefer to have the four in the handlers, but maybe I'm the only one to find more-readable :D Should be discussed with others maybe ?